basiclti/basiclti-portlet/src/java/org/sakaiproject/blti/ProviderServlet.java

```diff
@@ -789,6 +799,20 @@ private void loginUser(String ipAddress, User user) {
         sess.setUserEid(user.getEid());
     }
     
+    /*Checking if the email trusted consumer property and trusted consumer and not both enabled. 
+     * the case would be an error condition */
+    private boolean isEmailTrustedAndTrustedConsumerBothEnabled()  {
```

Comment:
I'd just delete this method (see above), but if there's a good reason to keep it, then you need to turn those strings into constants, instead of hard-coding them in multiple places.